mobly/controllers/android_device_lib/snippet_client_v2.py

```diff
@@ -83,15 +83,16 @@ class Config:
   """A configuration class for configuring the snippet client.
 
   Attributes:
-    am_instrument_options: dict[str, str], the Android am instrument options
-      used for controlling the `onCreate` process of the app under test. Note
-      that this should only be used for controlling the app launch process,
-      options for other purposes may not take effect and you should use snippet
-      RPCs. This is because Mobly snippet runner changes the subsequent
-      instrumentation process.
+    am_instrument_options: The Android am instrument options used for
```

Comment:
why remove the type indication?

Reply:
This is because I turned it into a dataclass attribute which requires type in code, so I deleted the type indication in docstring.